core/src/main/java/jenkins/util/xml/XMLUtils.java

```diff
@@ -137,16 +139,9 @@ public static void safeTransform(@Nonnull Source source, @Nonnull Result out) th
             throw new IllegalArgumentException(String.format("File %s does not exist or is not a 'normal' file.", file.getAbsolutePath()));
         }
 
-        FileInputStream fileInputStream = new FileInputStream(file);
```

Comment:
I was trying to keep the diff minimal... if you want I can optimize imports on the modified files, but that was re-sorting them for me with my current IDE and generating a large diff